src/01-functions.js

```diff
@@ -131,7 +148,13 @@ function compareTypes() {}
  *  flipSign(50);
  *  //> -50
  */
-function flipSign() {}
+function flipSign(a) {
```

Comment:
Nice use of Math.abs here and explicitly asking if the number is less than 0! Super slick! Lets make sure were consistent with our use of white space in our code and indentation though.